presto-main/src/main/java/com/facebook/presto/operator/HashBuilderOperator.java

```diff
@@ -152,7 +153,7 @@ private int getAndIncrementPartitionIndex(Lifespan lifespan)
     }
 
     @VisibleForTesting
-    public enum State
+    enum State
```

Comment:
nit: Usually we are trying to minimize the amount of changes to get something done. You can always create a refactoring commit to change something around.

In particular for these visibility changes to this class - you can either create a new commit, or you can leave them unchanged.

Reply:
sure